app/jobs/repository_import_event_job.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+# Documentation: https://developer.github.com/v3/activity/events/types/#repositoryimportevent
+class RepositoryImportEventJob < ApplicationJob
+  queue_as :porter_status
+
+  CREATE_COMPLETE = "Your GitHub repository was created."
+  IMPORT_FAILED = "We were not able to import starter code to your assignment, please try again."
+
+  def perform(payload_body)
+    github_repo_id = payload_body.dig("repository", "id")
+    status = payload_body.dig("status")
+    html_url = payload_body.dig("repository", "html_url")
+
+    repo = AssignmentRepo.find_by(github_repo_id: github_repo_id)
+    return if repo.blank?
+    # Group assignments coming in a follow up PR
+    # repo ||= GroupAssignmentRepo.find_by(github_repo_id: github_repo_id)
```

Comment:
If we go with this approach I will close #1548 and make this job work for `GroupAssignmentRepo`s too.